source/buildroot/Building_Buildroot_Image.rst

```diff
@@ -78,9 +99,22 @@ Build the Image
 With the configuration set, you can now build the image. This process compiles
 the necessary components and creates the root filesystem, kernel, and bootloader.
 
-.. code-block:: console
 
-   $ make TI_K3_BOOT_FIRMWARE_VERSION=11.00.05
+.. ifconfig:: CONFIG_part_variant in ('AM62X')
+
+   .. code-block:: console
+
+         $ make TI_K3_BOOT_FIRMWARE_VERSION=<version_tag>
+         For Example, version_tag will be 11.00.09 or 10.01.10
+         $ make TI_K3_BOOT_FIRMWARE_VERSION=11.00.09
+
+.. ifconfig:: CONFIG_part_variant in ('AM62LX')
+
+   .. code-block:: console
+
+         $ make TI_K3_BOOT_FIRMWARE_VERSION=<version_tag>
+         For Example, version_tag will be 11.00.05
+         $ make TI_K3_BOOT_FIRMWARE_VERSION=11.00.05
```

Comment:
```suggestion
.. list-table:: TI_K3_BOOT_FIRMWARE_VERSION

   * - Platform
     - Tag
   * - AM62X
     - 11.00.09
   * - AM62LX
     - 11.00.05

.. code-block:: console

   $ make TI_K3_BOOT_FIRMWARE_VERSION=<version_tag>
   For Example, version_tag will be 11.00.09 or 10.01.10
   $ make TI_K3_BOOT_FIRMWARE_VERSION=11.00.09
```

Nice job de-duplicating the previous section. This can also be applied to the other sections being added.